test/integration/image-component/valid-w3c-html/test/index.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env jest */
+
+import {
+  findPort,
+  killApp,
+  nextBuild,
+  nextStart,
+  waitFor,
+} from 'next-test-utils'
+import webdriver from 'next-webdriver'
+import validateHTML from 'html-validator'
+import { join } from 'path'
+
+const appDir = join(__dirname, '../')
+let appPort
+let app
+let browser
+
+describe('Image Component Valid W3C HTML', () => {
+  beforeAll(async () => {
+    await nextBuild(appDir)
+    appPort = await findPort()
+    app = await nextStart(appDir, appPort)
+    browser = await webdriver(appPort, '/')
+  })
+  afterAll(() => {
+    killApp(app)
+    browser = null
+  })
+
+  it('should be vaild W3C HTML', async () => {
+    await waitFor(1000)
+    expect(await browser.hasElementByCssSelector('img')).toBeTruthy()
+    const url = await browser.url()
+    const result = await validateHTML({
```

Comment:
We could add this to the existing image-component test suite [here](https://github.com/vercel/next.js/blob/canary/test/integration/image-component/default/test/index.test.js) instead and ensure the different layouts all validate correctly

Reply:
Good idea, done! :+1:

To keep things simple, I still added a new page `/valid-html-w3c` in the `default` folder.